### PR TITLE
Update Apache  License 2.0 and NOTICE

### PR DESCRIPTION
**Pull request by Apricity001**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ This command generates static content into the `build` directory and can be serv
 
 ## LICENSE
 
-Apache License 2.0
+[Apache License 2.0](./LICENSE).
```
Comment (caicancai) [inline README.md:29]: [Apache License, Version 2.0](LICENSE)

**Review by caicancai — CHANGES_REQUESTED** [review]

**Comment by caicancai** [comment]
cc @chaokunyang

**Review by caicancai — APPROVED** [review]
LGTM

**Review by chaokunyang — APPROVED** [review]
LGTM, thanks

**Pull request merged**